src/content/app/about/components/about-menu/AboutMenu.tsx

```diff
@@ -48,33 +51,53 @@ export const TopMenu = (props: Props) => {
 export const SideMenu = (props: Props) => {
```

Comment:
Can we remove this export as this is no more used outside this component?

Reply:
I'm removing the whole AboutMenu component. It isn't being used any longer. Thanks for pointing this out.